src/scene/vcPOI.cpp

```diff
@@ -1320,6 +1324,27 @@ void vcPOI::GenerateLineFillPolygon()
 
     m_pPolyModel->modelOffset = udDouble4x4::translation(m_line.pPoints[0]);
 
+    // Calculate area
+    m_meshArea = 0.0;
+    for (int i = 0; i < numIndices; i += 3)
+    {
+      int index1 = pIndices[i];
+      int index2 = pIndices[i + 1];
+      int index3 = pIndices[i + 2];
+
+      udDouble3 e1 = {
+        (double)(pVerts[index2].position.x - pVerts[index1].position.x),
+        (double)(pVerts[index2].position.y - pVerts[index1].position.y),
+        (double)(pVerts[index2].position.z - pVerts[index1].position.z) };
```

Comment:
This and the line below should like this:
```suggestion
      udDouble3 e1 = udDouble3::create(pVerts[index2].position - pVerts[index1].position);
```